infra/Pos/Reporting/Methods.hs

```diff
@@ -135,23 +134,22 @@ ipv4Local w =
 
 -- | Retrieves node info that we would like to know when analyzing
 -- malicious behavior of node.
-getNodeInfo :: (MonadIO m, MonadDiscovery m) => m Text
+getNodeInfo :: (MonadIO m) => m Text
 getNodeInfo = do
-    peers <- getPeers
     (ips :: [Text]) <-
         map show . filter ipExternal . map ipv4 <$>
         liftIO getNetworkInterfaces
-    pure $ sformat outputF (pretty $ listBuilderJSON ips) peers
+    pure $ sformat outputF (pretty $ listBuilderJSON ips)
   where
     ipExternal (IPv4 w) =
         not $ ipv4Local w || w == 0 || w == 16777343 -- the last is 127.0.0.1
-    outputF = ("{ nodeParams: '"%stext%"', otherNodes: "%listJson%" }")
```

Comment:
Ok, so yes, it's bad. I would like to see explanation of this change. Reports must include neighbors.